scripts/bootstrap-aio.yml

```diff
@@ -0,0 +1,147 @@
+  include: "{{ lookup('env', 'OA_DIR') | default('/opt/rpc-openstack/openstack-ansible', true) }}/tests/bootstrap-aio.yml"
+  vars:
+    openstack_user_config_overrides:
+      shared-infra_hosts:
+        aio1:
+          affinity:
+            galera_container: 3
+            rabbit_mq_container: 3
+          ip: 172.29.236.100
+
+- name: Execute the RPC-O AIO adjustments
+  hosts: localhost
+  user: root
+  tasks:
+    - name: Ensure that the env.d directory is present
+      file:
+        path: "/etc/openstack_deploy/env.d"
+        state: directory
+
+    # TODO(odyssey4me): Switch this to use https://review.openstack.org/423368
+    # once it is merged upstream and available in RPC-O.
+    - name: Rename standard OSA user-space configuration file
+      shell: |
+        result_code=0
+        if [ -e "/etc/openstack_deploy/user_variables.yml" ]; then
+          mv /etc/openstack_deploy/user_variables.yml /etc/openstack_deploy/user_osa_variables_overrides.yml
+          result_code=2
+        fi
+        exit ${result_code}
+      register: user_variables_rename
+      changed_when: user_variables_rename.rc == 2
+      failed_when: user_variables_rename.rc not in [0, 2]
+
+    # TODO(odyssey4me): Switch this to use https://review.openstack.org/423368
+    # once it is merged upstream and available in RPC-O.
+    - name: Rename standard OSA user-space secret file
+      shell: |
+        result_code=0
+        if [ -e "/etc/openstack_deploy/user_secrets.yml" ]; then
+          mv /etc/openstack_deploy/user_secrets.yml /etc/openstack_deploy/user_osa_secrets.yml
+          result_code=2
+        fi
+        exit ${result_code}
+      register: user_secrets_rename
+      changed_when: user_secrets_rename.rc == 2
+      failed_when: user_secrets_rename.rc not in [0, 2]
+
+    - name: Deploy the generally applicable RPC-O config files
+      copy:
+        src: "{{ rpco_cfg_src_path }}/{{ item.name }}"
+        dest: "/etc/openstack_deploy/{{ item.name }}"
+        mode: "{{ item.mode | default(omit) }}"
+      with_items:
+        - name: "user_osa_variables_defaults.yml"
+          mode: "0440"
+        - name: "user_rpco_variables_defaults.yml"
+          mode: "0440"
+        - name: "user_rpco_secrets.yml"
+        - name: "env.d/nova.yml"
+
+    - name: Deploy the RPC-O variable override file
+      template:
+        src: "{{ rpco_cfg_src_path }}/user_rpco_variables_overrides.yml.aio"
+        dest: "/etc/openstack_deploy/user_rpco_variables_overrides.yml"
+
+    - name: Deploy the RPC-O ceph env.d file
+      config_template:
+        src: "{{ rpco_cfg_src_path }}/env.d/ceph.yml"
+        dest: "/etc/openstack_deploy/env.d/ceph.yml"
+        config_overrides:
+          container_skel:
+            ceph_osd_container:
+              properties:
+                is_metal: false
+        config_type: "yaml"
+      when: "{{ rpco_deploy_ceph | bool }}"
+
+    - name: Deploy the RPC-O ceph conf.d file
+      copy:
```

Comment:
move this into the task at line 77 and add a conditional to the dict for each item